docs/design/0021-two-way-github-sync/two-way-github-sync.md

```
@@ -0,0 +1,72 @@
+---
+
+# Two-way GitHub Sync
+
+Source backlog item: `docs/method/backlog/up-next/PROCESS_two-way-github-sync.md`
+Legend: PROCESS
+
+## Sponsors
+
+- Human: Repository Operator
+- Agent: System Automator
+
```

markdownlint-cli2 (0.22.0)

[warning] Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)